packages/manager/modules/exchange/src/external-contact/external-contact.html

```diff
@@ -2,221 +2,120 @@
-                            data-ng-if="!ctrl.filter"
-                        >
-                            <span
-                                class="fa fa-search"
-                                aria-hidden="true"
-                            ></span>
-                        </span>
-                    </div>
-                </div>
-            </form>
-
-            <div class="table-responsive">
-                <table class="table table-hover">
-                    <thead>
-                        <tr>
-                            <th
-                                scope="col"
-                                data-translate="exchange_tab_EXTERNAL_CONTACTS_table_headers_externalEmailAddress"
-                            ></th>
-                            <th
-                                scope="col"
-                                data-translate="exchange_tab_EXTERNAL_CONTACTS_table_headers_displayName"
-                            ></th>
-                            <th
-                                class="text-center"
-                                scope="col"
-                                data-translate="exchange_tab_EXTERNAL_CONTACTS_table_headers_creationDate"
-                            ></th>
-                            <th
-                                class="text-center"
-                                scope="col"
-                                data-translate="exchange_tab_EXTERNAL_CONTACTS_table_headers_hiddenFromGAL"
-                            ></th>
-                            <th
-                                class="text-center"
-                                scope="col"
-                                data-translate="exchange_tab_EXTERNAL_CONTACTS_table_headers_state"
-                            ></th>
-                            <th class="min-width text-center"></th>
-                        </tr>
-                    </thead>
-
-                    <tbody data-ng-if="ctrl.contactsLoading">
-                        <tr>
-                            <td colspan="6" class="text-center">
-                                <oui-spinner></oui-spinner>
-                            </td>
-                        </tr>
-                    </tbody>
+            </oui-column>
+            <oui-column
+                data-title="'exchange_tab_EXTERNAL_CONTACTS_table_headers_displayName' | translate"
+                property="displayName"
+            >
+            </oui-column>
+            <oui-column
+                data-title="'exchange_tab_EXTERNAL_CONTACTS_table_headers_creationDate' | translate"
+                property="creationDate"
+                type="date"
+            >
+                <span data-ng-bind="$value | date:'mediumDate'"></span>
+            </oui-column>
+            <oui-column
+                data-title="'exchange_tab_EXTERNAL_CONTACTS_table_headers_hiddenFromGAL' | translate"
+                property="hiddenFromGAL"
+            >
+                <span
```

Comment:
No alternative content for `sr-only` ?

Reply:
I will remove the `aria-hidden` attribute (not needed).

Refer to: https://ovh-ux.github.io/ovh-ui-kit-documentation/#!/oui-angular/tooltip#accessibility